app/controllers/ems_common.rb

```diff
@@ -694,6 +700,9 @@ def get_form_vars
         @edit[:new][:port] = @ems.port ? @ems.port : ManageIQ::Providers::OpenshiftEnterprise::ContainerManager::DEFAULT_PORT
       elsif params[:server_emstype] == ManageIQ::Providers::AtomicEnterprise::ContainerManager.ems_type
         @edit[:new][:port] = @ems.port ? @ems.port : ManageIQ::Providers::AtomicEnterprise::ContainerManager::DEFAULT_PORT
+      elsif params[:server_emstype] == "vmware_cloud"
```

Comment:
Thanks. I must have tried too many things while making sure values are properly propagated. I will wait for your comment on the generic block above and then push again.